### PR TITLE
Updated licensing description

### PR DESCRIPTION
**Pull request by billmcilhargey**
Based on recent licensing changes, adjusted licensing verbage

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/compliance-manager-quickstart.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ description: "Use the Compliance Manager quickstart guide to help you along your
 
 **In this article:** Use this quickstart guide to help you along your journey of using Microsoft Compliance Manager to manage your organization’s compliance with regulations, policies, and standards.
 
-Compliance Manager provides intelligent and actionable data upon your first visit. Compliance Manager also has advanced capabilities for scaling your compliance when you’re ready. All customers can use the [Data Protection Baseline assessment](compliance-manager-assessments.md#data-protection-baseline-default-assessment) to ensure you're protecting critical data. With Microsoft 365 E5, you can use over 150 pre-built assessments to help you comply with regulations across your industry and region, and use custom assessments to extend Compliance Manager across all of your assets.
+Compliance Manager provides intelligent and actionable data upon your first visit. Compliance Manager also has advanced capabilities for scaling your compliance when you’re ready. Available assessments depend on your licensing agreement; learn more(https://go.microsoft.com/fwlink/?linkid=2132371).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-quickstart.md:23]:
```suggestion
Compliance Manager provides intelligent and actionable data upon your first visit. Compliance Manager also has advanced capabilities for scaling your compliance when you’re ready. Available assessments depend on your licensing agreement; to learn more, see [Microsoft 365 licensing guidance for security & compliance](https://go.microsoft.com/fwlink/?linkid=2132371).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/compliance/compliance-manager-quickstart.md (hunk 1 above)
Comment (chvukosw) [reply microsoft-365/compliance/compliance-manager-quickstart.md:23]: This is approved.

Comment (chvukosw) [reply microsoft-365/compliance/compliance-manager-quickstart.md:23]: This is approved.

**Comment by yogkumgit** [comment]
@computeronix

**Comment by chvukosw** [comment]
I approve of this. I don't see where else I need to indicate it. Please proceed, thanks.

**Review by chvukosw — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@chvukosw - please merge the commit on the copy/edit review done by myself or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**